x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -54,7 +54,7 @@
                         MetadataUtils.DEFAULT_RESERVED_METADATA))
                 .put("kibana_admin", kibanaAdminUser("kibana_admin", MetadataUtils.DEFAULT_RESERVED_METADATA))
                 .put("kibana_user", kibanaAdminUser("kibana_user",
-                    MetadataUtils.getDeprecatedReservedMetadata("Please use the [kibana_admin] role instead")))
+                    MetadataUtils.getDeprecatedReservedMetadata("Use kibana_admin instead.")))
```

Comment:
nitpick: The square brackets are used pretty consistently to denote literal values. Probably better to keep them.

Reply:
Thanks @ywangd - @gchaps how do you feel about reintroducing the square brackets around `kibana_admin`?